### PR TITLE
Adding jupyterlab_gather

### PR DESCRIPTION
**Pull request by gjmooney**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/example/meta.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ build:
   # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
```
Comment (synapticarbors) [inline recipes/example/meta.yml:33]: Please do not modify this file

File: recipes/jupyterlab-gather/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_gather-{{ version }}.tar.gz
+  sha256: 1246a16edb896676bff19e2ec8e05779cbb7fdfb7af30e0fba894d2bc4a0e2c2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/jupyterlab-gather/meta.yaml:18]: It looks like you may need `hatch-jupyter-builder >=0.5` in the `host` section.

File: recipes/jupyterlab-gather/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    - python >=3.8
+    - hatchling >=1.5.0
+    - hatch-nodejs-version >=0.3.2
+    - pip
+  run:
```
Comment (synapticarbors) [inline recipes/jupyterlab-gather/meta.yaml:23]: It looks like you probably need `jupyter_server >=2.0.1,<3` under the `run` dependencies.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/jupyterlab-gather/meta.yaml (hunk 2 above)
Comment (gjmooney) [reply recipes/jupyterlab-gather/meta.yaml:18]: Thanks!

File: recipes/jupyterlab-gather/meta.yaml (hunk 3 above)
Comment (gjmooney) [reply recipes/jupyterlab-gather/meta.yaml:24]: Thanks again!

File: recipes/example/meta.yml (hunk 1 above)
Comment (gjmooney) [reply recipes/example/meta.yml]: Fixed, thanks!

**Comment by gjmooney** [comment]
I confirm that I'm willing to be listed as a maintainer.

@conda-forge-admin, please ping conda-forge/help-python Ready for review!

**Review by jaimergp — APPROVED** [review]

File: recipes/jupyterlab-gather/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: 1246a16edb896676bff19e2ec8e05779cbb7fdfb7af30e0fba894d2bc4a0e2c2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
```
Comment (jaimergp) [inline recipes/jupyterlab-gather/meta.yaml:19]:
```suggestion
    - python {{ python_min }}
```

File: recipes/jupyterlab-gather/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - hatchling >=1.5.0
+    - hatch-nodejs-version >=0.3.2
+    - hatch-jupyter-builder >=0.5
+    - pip
+  run:
+    - python >=3.8
```
Comment (jaimergp) [inline recipes/jupyterlab-gather/meta.yaml:25]:
```suggestion
    - python >={{ python_min }}
```

File: recipes/jupyterlab-gather/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    - jupyter_server >=2.0.1,<3
+    - pyjwt
+
+test:
+  imports:
+    - jupyterlab_gather
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (jaimergp) [inline recipes/jupyterlab-gather/meta.yaml:35]:
```suggestion
    - pip
    - python {{ python_min }}
```

File: recipes/jupyterlab-gather/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
+{% set name = "jupyterlab-gather" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_gather-{{ version }}.tar.gz
```
Comment (jaimergp) [inline recipes/jupyterlab-gather/meta.yaml:9]:
```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_gather-{{ version }}.tar.gz
```

**Pull request merged**